oio/api/base.py

```diff
@@ -63,7 +57,7 @@ def __init__(self, endpoint=None, pool_manager=None,
         if not pool_manager:
             pool_manager_conf = {k: int(v)
                                  for k, v in iteritems(kwargs)
-                                 if k in _POOL_MANAGER_OPTIONS_KEYS}
+                                 if k in URLLIB3_POOLMANAGER_KWARGS}
```

Comment:
Ah, it took me a while to understand why tests failed here with that:
```
ERROR: tests.functional.api.test_objectstorage_performance.TestObjectStorageApiPerformance.test_object_create_32_md5_checksum
----------------------------------------------------------------------
_StringException: Traceback (most recent call last):
  File "/home/travis/build/open-io/oio-sds/tests/functional/api/test_objectstorage_performance.py", line 28, in setUp
    source_address=('127.0.0.8', 0))
  File "oio/api/object_storage.py", line 110, in __init__
    self.directory = DirectoryClient(conf, logger=self.logger, **kwargs)
  File "oio/directory/client.py", line 30, in __init__
    **kwargs)
  File "oio/common/client.py", line 83, in __init__
    endpoint='/'.join(ep_parts), service_type='proxy', **kwargs)
  File "oio/api/base.py", line 59, in __init__
    for k, v in iteritems(kwargs)
  File "oio/api/base.py", line 60, in <dictcomp>
    if k in URLLIB3_POOLMANAGER_KWARGS}
TypeError: int() argument must be a string or a number, not 'tuple'
```

The reason is the old list only allowed integer arguments and using this straight from URLLIB3_POOLMANAGER_KWARGS is more liberal.
Looking at git log, python3 compat might have required the `int()` cast? but 7966a1e63c2850f3e5c2faa494d2d0f9ff4adf26 does not say why limit to int args and not pass-through non-int args, and `get_pool_manager` now does the cast and filtering itself...

So I'll just drop the whole filtering back out and assume the function does it properly, and see how tests hold up.